index.ts

```diff
@@ -821,12 +821,12 @@ commander
               manifestFileName: PROJECT_MANIFEST_BASENAME,
               description,
             },
-          }, {}, (err: any, { data }: any) => {
+          }, {}, (err: any, response: any) => {
```

Comment:
So, when it returns an `err`, it actually returns `undefined` as the response, so when using `{ data }` you get an unhandled promise rejection error. This fixes that.